peak_core/peak_core.py

```diff
@@ -199,6 +199,7 @@ def __init__(self, peak_generator):
         self._setup_config()
 
     def get_config_bitstream(self, instr):
+        print("INSTR=",instr, type(instr) == self.wrapper.instruction_type())
```

Comment:
let's not include this debugging logging

Reply:
Just removed